### DISCUSSION TITLE
how to setup openVinoToolkit  in my local machine

### DISCUSSION THREAD

**Discussion by Mac16661** [openvinotoolkit/openvino#20747] [Q&A] [answered]
Can anybody provide some guide how to setup this project in local dev env.

 And I am using AMD machine will that be any problem ? 

I am a complete beginner to this so it will grate if anybody help me.

Thanks in advance.

**Comment by ilya-lavrenov** [comment, marked as answer]
Hi @Mac16661 
1. Start with installation guide https://docs.openvino.ai/2023.1/openvino_docs_install_guides_installing_openvino_pip.html
2. Try to run OpenVINO Python Notebooks https://docs.openvino.ai/2023.1/notebooks_installation.html, which are ready to run AI use cases, which will work on AMD CPUs as well.
